app/assets/javascripts/discourse/app/controllers/users.js

```diff
@@ -57,4 +65,9 @@ export default Controller.extend({
   loadMore() {
     this.model.loadMore();
   },
+
+  @action
+  updateGroupParam(selectedGroups, currentSelection) {
+    this.set("group", currentSelection ? currentSelection.name : null);
```

Comment:
Is there a reason we use the group name instead of just the id here? That way, we get to avoid the `find`ing in `selectedGroupId` all together.

Reply:
The backend accepts group name as a parameter so we can't use id